lib/presentation/crowd_action/pages/crowd_action_details.dart

```diff
@@ -0,0 +1,267 @@
+
+class CrowdActionDetailsPage extends StatelessWidget {
+  final CrowdAction crowdAction;
+
+  const CrowdActionDetailsPage({
+    Key? key,
+    required this.crowdAction,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: NestedScrollView(
+        headerSliverBuilder: (BuildContext context, bool innerBoxIsScrolled) {
+          return <Widget>[
+            SliverAppBar(
+              expandedHeight: 310.0,
+              pinned: true,
+              backgroundColor: kAccentColor,
+              automaticallyImplyLeading: false,
+              title: Row(
+                children: <Widget>[
+                  ElevatedButton(
+                    onPressed: () => context.router.pop(),
+                    style: ElevatedButton.styleFrom(
+                      shape: const CircleBorder(),
+                      primary: Colors.white,
+                      onPrimary: kAccentColor,
+                    ),
+                    child: const Icon(
+                      Icons.chevron_left,
+                      color: kPrimaryColor200,
+                    ),
+                  ),
+                  const Expanded(
+                    child: SizedBox(),
+                  ),
+                  ElevatedButton(
+                    onPressed: () {},
+                    style: ElevatedButton.styleFrom(
+                      shape: const CircleBorder(),
+                      primary: kAccentColor,
+                      onPrimary: kAccentColor,
+                    ),
+                    child: const Icon(Icons.upload, color: Colors.white),
+                  )
+                  // Your widgets here
+                ],
+              ),
+              flexibleSpace: FlexibleSpaceBar(
+                background: Image.network(
+                  crowdAction.image ?? "",
+                  fit: BoxFit.cover,
+                ),
+              ),
+            ),
+          ];
+        },
+        body: SingleChildScrollView(
+          child: Column(
+            children: [
+              Container(
+                color: kAlmostTransparent,
+                padding:
+                    const EdgeInsets.symmetric(horizontal: 20, vertical: 30),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      crowdAction.name,
+                      style: Theme.of(context)
+                          .textTheme
+                          .headline4
+                          ?.copyWith(fontWeight: FontWeight.bold),
+                    ),
+                    const SizedBox(
+                      height: 20,
+                    ),
+                    Wrap(
+                      spacing: 12.0,
+                      children: [
+                        SignUpChip(),
+                        if (crowdAction.tags?.isNotEmpty == true)
```

Comment:
Maybe will have a category (and sub-category instead).
👉 Check if the design would change in that case.

Reply:
Nope, it wouldn't change